beet/resources/pack_format_registry.json

```diff
@@ -0,0 +1,5112 @@
+[
+  {
+    "id": "1.21.6",
+    "name": "1.21.6",
+    "release_target": null,
+    "type": "release",
+    "stable": true,
+    "data_version": 4435,
+    "protocol_version": 771,
+    "data_pack_version": 80,
+    "resource_pack_version": 63,
+    "build_time": "2025-06-17T11:07:28+00:00",
+    "release_time": "2025-06-17T11:10:28+00:00",
+    "sha1": "ac29333211286b32d68ca4a16d2026a834ac1430"
+  },
+  {
+    "id": "1.21.6-rc1",
+    "name": "1.21.6 Release Candidate 1",
+    "release_target": null,
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 4434,
+    "protocol_version": 1073742080,
+    "data_pack_version": 80,
+    "resource_pack_version": 63,
+    "build_time": "2025-06-12T12:01:32+00:00",
+    "release_time": "2025-06-12T12:04:37+00:00",
+    "sha1": "8235a361129024241eaa6a4bf0347ef1f0427cd0"
+  },
+  {
+    "id": "1.21.6-pre4",
+    "name": "1.21.6 Pre-Release 4",
+    "release_target": null,
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 4433,
+    "protocol_version": 1073742079,
+    "data_pack_version": 80,
+    "resource_pack_version": 63,
+    "build_time": "2025-06-10T12:19:37+00:00",
+    "release_time": "2025-06-10T12:22:36+00:00",
+    "sha1": "8ecc82c51629ed912439bcab716a8af29670f45a"
+  },
+  {
+    "id": "1.21.6-pre3",
+    "name": "1.21.6 Pre-Release 3",
+    "release_target": null,
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 4432,
+    "protocol_version": 1073742078,
+    "data_pack_version": 80,
+    "resource_pack_version": 63,
+    "build_time": "2025-06-04T13:30:24+00:00",
+    "release_time": "2025-06-04T13:33:25+00:00",
+    "sha1": "e1f07a9baa33f968aa4046ee0f246b6505e5a366"
+  },
+  {
+    "id": "1.21.6-pre2",
+    "name": "1.21.6 Pre-Release 2",
+    "release_target": null,
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 4431,
+    "protocol_version": 1073742077,
+    "data_pack_version": 79,
+    "resource_pack_version": 63,
+    "build_time": "2025-06-02T13:37:41+00:00",
+    "release_time": "2025-06-02T13:40:47+00:00",
+    "sha1": "42a30ad342b7fd2502f1be8527d65ea74f10ae8e"
+  },
+  {
+    "id": "1.21.6-pre1",
+    "name": "1.21.6 Pre-Release 1",
+    "release_target": null,
+    "type": "snapshot",
+    "stable": false,
+    "data_version": 4430,
+    "protocol_version": 1073742076,
+    "data_pack_version": 79,
+    "resource_pack_version": 63,
+    "build_time": "2025-05-28T09:31:08+00:00",
+    "release_time": "2025-05-28T09:34:04+00:00",
+    "sha1": "72f3aee6de4c96f87cf0994b0fd0e25737034acb"
+  },
+  {
+    "id": "25w21a",
+    "name": "25w21a",
+    "release_target": null,
+    "type": "snapshot",
```

Comment:
Is it necessary (or even a good idea) to support snapshots? Mecha and other beet features might not necessarily support these versions. Maybe we should just hardcode a list of pack formats for all the release versions?